experimental/cloudstrapper/playbooks/roles/control/templates/main.tf.1.3.org

```diff
@@ -0,0 +1,87 @@
+################################################################################
```

Comment:
I don't see the point of these `.org` files. Could you explain why we need these?

Reply:
I left the .org files to indicate what a base version of the file looks like for the user to compare. I will remove these files.